lib/crystalball/rspec/runner/configuration.rb

```diff
@@ -66,7 +66,7 @@ def repo_path
         attr_reader :values
 
         def run_requires
-          self['requires'].each { |f| require f }
+          Array(self['requires']).each { |f| require f }
```

Comment:
can you pass array above? Instead of wrapping to array here?

Reply:
Can't do that with ENV